neo/rawio/intanrawio.py

```diff
@@ -335,12 +343,21 @@ def _get_signal_size(self, block_index, seg_index, stream_index):
             mask = self.header["signal_channels"]["stream_id"] == stream_id
             signal_channels = self.header["signal_channels"][mask]
             channel_ids = signal_channels["id"]
-            channel_id_0 = channel_ids[0]
-            size = self._raw_data[channel_id_0].size
+
+            stream_name = self.header["signal_streams"][stream_index]["name"][:]
+            stream_is_digital = stream_name in digital_stream_names
+            if stream_is_digital:
+                field_name = name_to_field_name_digital[stream_name]
```

Comment:
What is a field name? 

I wonder if we will remember this in the future. It is not:
1) Neo terminology (which we are bound to here--block, stream, segment)
2)  Not clear to me as a word in this case

Reply:
This is standard terminology for struct and structured arrrays in numpy:

https://numpy.org/doc/stable/glossary.html#term-field

Reply:
Okay perfect! You're right. I never use structured arrays, but Neo and SI use them all the time, so I should get more used to the naming. Maybe it should be 

```
stream_name_to_field_name_digital
```

To be as explicit as possible?

Reply:
Good idea. Let's start with this.

Reply:
I decided what I think is a simpler approach. See below.